sync/config.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/coveooss/credentials-sync/credentials"
 	"github.com/coveooss/credentials-sync/logger"
 	"github.com/coveooss/credentials-sync/targets"
+	"github.com/hashicorp/go-multierror"
```

Comment:
Gives us a struct `multierror` that makes it possible to aggregate errors and that pretty print them for us.
e.g.
```
// printing
print(aMultiErrorWith4Errors)
// output 
4 errors occurred:
        * lol une erreur ici
        * lol une erreur ici
        * lol une erreur ici
        * lol une erreur ici
```